src/t2iapi/device/service.proto

```diff
@@ -66,6 +66,17 @@ service DeviceService {
     rpc GetRemovableDescriptors (google.protobuf.Empty)
         returns (t2iapi.device.GetRemovableDescriptorsResponse);
 
+    /*
+    Get all descriptor handles which can be removed and reinserted into the MDIB of the device
+    and whose descriptors are of the given type.
+    Includes handles which have been removed already and can be reinserted.
+    Handles must stay the same once reinserted into the MDIB.
+    The handles shall be representative of the devices capabilities to remove
```

Comment:
This sentence needs adjusting as it only applies to the abstract case.

Reply:
Done.